package.json

```diff
@@ -7,11 +7,11 @@
   "types": "types/index.d.ts",
   "scripts": {
     "lint": "standard",
-    "mariadb": "docker run -d -p 3306:3306 -e MYSQL_ALLOW_EMPTY_PASSWORD=yes --rm mariadb:10.1",
+    "mariadb": "docker run -d -p 3307:3307 -e MYSQL_ALLOW_EMPTY_PASSWORD=yes --rm mariadb:10.1",
```

Comment:
Why the port changes here?

Reply:
oops... was having issues with my local mariadb running on the same port and so i just changed it in the code to get the test to pass.. forgot to change it back.. I can fix this.